### PR TITLE
ASC-472 Added bcrypt for password encryption

### PR DESCRIPTION
**Pull request by AkashExcelra**
Integrated password encryption for new user and cross validate with database, also updated test cases for validation of login

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,8 +11,9 @@
     "prepare": "husky"
   },
   "devDependencies": {
-    "react": "^18.3.1",
+    "@types/bcrypt": "^5.0.2",
```
Comment (himanshubexcelra) [inline package.json:14]: Why this is upated?

File: apps/api/.eslintrc.js
[the remark this answers is not in the document]
Comment (AkashExcelra) [reply apps/api/.eslintrc.js:1]: Issue resolve Commit with initial commit changes

File: aidd_syncoop.sql
[the remark this answers is not in the document]
Comment (AkashExcelra) [reply aidd_syncoop.sql:1]: Issue resolved - File removed

File: apps/web/app/globals.css [hunk 2]
```diff
@@ -51,12 +51,12 @@
 }
 
 
-@media (prefers-color-scheme: dark) {
+/* @media (prefers-color-scheme: dark) {
```
[the remark this answers is not in the document]
Comment (AkashExcelra) [reply apps/web/app/globals.css]: Issue resolved - Undo changes

File: apps/web/.eslintrc.js
[the remark this answers is not in the document]
Comment (AkashExcelra) [reply apps/web/.eslintrc.js:1]: Issue resolve Commit with initial commit changes

File: package.json (hunk 1 above)
Comment (AkashExcelra) [reply package.json]: Issue resolve Commit with initial commit changes

File: apps/api/prisma/schema.prisma
[the remark this answers is not in the document]
Comment (AkashExcelra) [reply apps/api/prisma/schema.prisma:1]: Issue resolve Commit with initial commit changes

**Pull request merged**